src/components/html.cjsx

```diff
@@ -12,3 +12,10 @@ module.exports = React.createClass
       <div className={classes.join(' ')} dangerouslySetInnerHTML={__html:@props.html} />
     else
       <span className={classes.join(' ')} dangerouslySetInnerHTML={__html:@props.html} />
+
+  componentDidMount: ->
```

Comment:
Can you include $ = require 'jquery'?

I've had a preference for not including jQuery unless necessary (so far it was only used for `$.ajax`) especially since we are only targeting modern browsers. we can chat about it though ; )